tests/framework/PostProcessors/LimitSurface/tests

```diff
@@ -27,14 +27,18 @@
   rel_err = 0.0001
  [../]
 
- [./testLimitSurfaceIntegralPPWihtBoundingError]
+ [./testLimitSurfaceIntegralPPWithBoundingError]
   type = 'RavenFramework'
   input = 'test_LimitSurface_with_err_bounds.xml'
   output = 'limitSurfaceBoundingError/LimitSurfaceWeightedPb_dump.xml'
-  csv = 'limitSurfaceBoundingError/LimitSurfaceWeightedPb_dump.csv'
   max_time = 300
   rel_err = 0.001
```

Comment:
The real test of this is if we can make the rel_err smaller. (That said, that can definitely be a separate pull request.)